src/zope/interface/interfaces.py

```diff
@@ -997,19 +997,62 @@ def subscribe(required, provided, subscriber): # pylint:disable=arguments-differ
            Subscribers have no names.
         """
 
+    def subscribed(required, provided, subscriber):
+        """
+        Check whether the object *subscriber* is registered directly
+        with this object via a previous
+        call to ``subscribe(required, provided, subscriber)``.
```

Comment:
Word-wrapping feels a little bit weird here.  Any particular reason?

Reply:
The double-backtick block wraps as a single entity.

Reply:
Yes, but `call to` is outside the double-ticked block and would easily fit on the previous line.

Reply:
I just realized that's what you meant and was committing a change as you commented :)